samples/samples/connection_sample.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+
+from google.cloud import spanner
+from google.cloud import spanner_dbapi
+
+from google.cloud.spanner_dbapi import connection, InterfaceError
+
+
+def list_all():
+    spanner_client = spanner.Client()
+
+    for inst in spanner_client.list_instances():
+        print(inst.name)
+        for db in inst.list_databases():
+            print(db.name)
+
+
+def create_connection(instance_id, database_id):
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+
+    print("Connection is created.")
+
+
+def get_cursor(instance_id, database_id):
+    # Opened connection
+    my_connection = spanner_dbapi.connect(instance_id, database_id)
+    cursor = my_connection.cursor()
+    print("Got cursor.")
+
+    # Closed connection
+    my_connection.close()
+    try:
```

Comment:
Why `try-except`? We don't actually need error causing cases in samples. We're showing how to use API, not how to get an error.

Reply:
The idea was to show that it is impossible to get cursor in closed connection.

Reply:
No, no, we're not showing error cases. We should show the right way of use instead.